src/main/kotlin/app/revanced/utils/adb/Adb.kt

```diff
@@ -34,7 +34,7 @@ internal class Adb(
 
             PackageManager(device).install(file)
         } else {
-            logger.info("Installing by mounting")
+            logger.info("Installing by mounting (method: ${if (Constants.IS_SUPERSU) "SuperSU" else "MagiskSU"})")
```

Comment:
"Method" is not the correct terminology, maybe chose something else.

Reply:
Changed it from `method` -> `root type` as it makes more sense in this context.